tests/oclc_/test_title_author_lookup_classify.py

```diff
@@ -16,7 +16,7 @@
 )
 
 from oclc.classify import (
-    OCLCXMLParser,
```

Comment:
Note to Amy: this file is just reflecting the new name of the OCLCTitleAuthorLookupXMLParser.